action.yaml

```diff
@@ -25,6 +25,31 @@ inputs:
     required: true
     default: 'master'
 
+  git_checkout_extra_args:
+    description: 'Any extra args to pass to `git checkout` like --recurse-submodules (only used when current branch is not target_branch), default = ""'
+    required: false
+    default: ''
+
+  git_fetch_extra_args:
+    description: 'Any extra args to pass to `git fetch` like --recurse-submodules, default = ""'
+    required: false
+    default: ''
```

Comment:
Also, I'd like you to remove 'extra' from the var names to slim them down a little. Less to type, fewer potential errors.

Reply:
done